src/scripts/battleFrontier/BattleFrontierBattle.ts

```diff
@@ -56,6 +56,7 @@ class BattleFrontierBattle extends Battle {
         } else {
             this.enemyPokemon(null);
         }
+        player.lowerItemMultipliers(MultiplierDecreaser.Battle);
```

Comment:
Battle frontier shouldn't decrease normal shop prices, just the BF shop prices.

Reply:
Good point, I reverted the battle frontier change as the shop already has the price reduction in the town.ts file.